iosHyperskillApp/iosHyperskillApp/highlight.css

```diff
@@ -57,6 +57,20 @@ pre code::-webkit-scrollbar-thumb-track {
     border-radius: 0.375rem;
 }
 
+pre code>span {
+    counter-increment: line;
+}
+
+pre code>span::before {
+    display: inline-block;
+    width: 1rem;
+    padding-right: 0.5rem;
+    margin-right: 0.5rem;
+    text-align: right;
+    content: counter(line);
+    border-right: 1px solid var(--black25);
```

Comment:
We should achieve a similar appearance

<img width="405" alt="Screenshot 2023-02-08 at 12 20 20 AM" src="https://user-images.githubusercontent.com/8586063/217317444-15eae52b-d529-4910-9c19-6c0325c294d7.png">

1. The gutter line color should support light and dark modes. Move `border-right` to the `@media screen` and set the border color `#0000001f` for light mode and `#ffffff1f` for dark mode.
2. Provide text color for the line number, now it's completely black. We can use our branded color `color_on_surface_alpha_38`, also it needs support for light and dark modes.